enterprise/coderd/prebuilds/membership_test.go

```diff
@@ -32,87 +38,168 @@ func TestReconcileAll(t *testing.T) {
 	}
 
 	tests := []struct {
-		name                  string
-		includePreset         bool
-		preExistingMembership bool
+		name                       string
+		includePreset              []bool
+		preExistingOrgMembership   []bool
```

Comment:
I'm a bit confused here, why are we now using a slice of bool in these cases? Could you add a comment here as well as in the test cases?

Reply:
The slices of bool allow us to test all the combinations of these scenarios. so in two "test cases", we actually technically have 16 tests.